src/server/db_slice.cc

```diff
@@ -416,6 +421,12 @@ auto DbSlice::FindReadOnly(const Context& cntx, string_view key, unsigned req_ob
 }
 
 pair<PrimeIterator, ExpireIterator> DbSlice::FindExt(const Context& cntx, string_view key) const {
+  return FindInternal(cntx, key, FindInternalMode::kUpdateStats);
```

Comment:
also here kDontUpdateStats? Only FindReadOnly should have the kUpdateStats I think

Reply:
`FindExt()` should be changed soon, but it can be used for both read-only and modify operations. Like `Find()` was before I added `FindReadOnly()`.
Since it's causing some tests to fail (and changes existing behavior), let's leave it as is for now and revisit in the other PR...